### ISSUE TITLE
@load only loads the first DataFrame

### ISSUE THREAD

**Issue by tcovert** [JuliaIO/JLD.jl#96]
I've recently noticed (since I upgraded to 0.5) that `@load` only loads the first DataFrame in a JLD file containing multiple DataFrames.  Here are [two](https://dl.dropboxusercontent.com/u/975657/well_list.jld) [example](https://dl.dropboxusercontent.com/u/975657/scout_tickets.jld) JLD's with this property.  The obvious work-around is to use the `load()` function, which does work, but it would be nice if the macro syntax worked as well.

**Comment by tcovert** [comment]
If it helps, here is my `versioninfo()`:
```
julia> versioninfo()
Julia Version 0.5.0
Commit 3c9d753* (2016-09-19 18:14 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin15.6.0)
  CPU: Intel(R) Core(TM) i7-4578U CPU @ 3.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
```
and `Pkg.status("JLD")`:
```
julia> Pkg.status("JLD")
 - JLD                           0.6.3
```

**Comment by simonster** [comment]
This should have been fixed by #88. Can you still reproduce?

**Comment by simonster** [comment]
In the absence of a response, I am assuming this is now fixed.
